### PR TITLE
Added coverage badge

### PR DESCRIPTION
**Pull request by surajnarwade**
Added coverage badge as well as coverage test in Makefile.

### CODE REVIEW

**Comment by surajnarwade** [comment]
@kadel , I need to turn on coverage on coveralls, right ?
bit confused here

File: Makefile [hunk 1]
```diff
@@ -56,7 +56,7 @@ test-unit:
 
 # Run all tests
 .PHONY: test
-test: test-dep check-vendor validate test-unit
+test: test-dep check-vendor validate test-unit test-unit-cover
```
Comment (kadel) [inline Makefile]: This runs tests twice.

**Comment by kadel** [comment]
It might be better to create new Makefile rule called `travis-tests`

that will look like this:

```Makefile
# Tests that are run on travs-ci
.PHONY: travis-tests
travis-tests: test-dep check-vendor validate test-unit-cover
```

This rule will be used in `.travis.yaml` instead of `make test`.

We can leave `make test` as it was, so it can be easily run by Kedge developers on their local machine.

**Comment by kadel** [comment]
I've disabled coveralls comments in the coveralls settings. It's more annoying than useful :-)

**Pull request merged**